### PR TITLE
Feature/add l and d guide

### PR DESCRIPTION
**Pull request by RobjS**
The intention of this re-write is to:

* Help people find ideas on how to use their L&D allowance
* Make the process for using your L&D allowance clearer
* Make the process less inimidating

### CODE REVIEW

File: guides/learning-and-development.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: Learning and development
+---
+
+We have an annual learning and development allowance of £750 and up to 4 days
```
Comment (deberny) [inline guides/learning-and-development.md:5]: Should we update this to '£1000' ?

File: guides/learning-and-development.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+per person.
+
+You can use this for professional and personal development activities like
+workshops, short courses or conferences.
```
Comment (deberny) [inline guides/learning-and-development.md:9]: I wonder if we can add learning platform subscriptions. I know few people that use the budget to subscribe to training platforms 'Skills Matter' or 'Linda..'
And if it's a wellbeing thing can a person use part of the budget for example 'yoga subscription or meditation app' ?
I might be going too wild here..

File: guides/learning-and-development.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+### Book the activity
+
+- Agree the plan with your line manager before you book anything
+- Log it in [Breathe](https://hr.breathehr.com/) as a training request:
+  - Go to "My profile" on the top menu
+  - Click the "Training" button
+  - Click the "+" button to add a new request
+  - Include weblinks and an estimate of the ticket, accommodation and travel
+    costs (up to the maximum of £750, though you are welcome to pay the
```
Comment (deberny) [inline guides/learning-and-development.md:42]: £1000

File: guides/learning-and-development.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    difference for more expensive events)
+- Think about sharing event with the rest of your team if it's something other
+  people might find relevant/useful
+- Make sure your project team know when you're going to be away
```
Comment (deberny) [inline guides/learning-and-development.md:46]: I think there is an additional step in breathe here. Apart from logging training. You also need to add 'Leave' request with the category training, so it updates the scheduling calendar.

Comment (deberny) [reply guides/learning-and-development.md:46]: Perhaps add 'It also useful when you add your training days into your calendar for others to know' or something like that

**Review by deberny — COMMENTED** [review]
Looks good to me. The step by step process is very clear and helpful.
I've added few ideas you might want to discuss with Vee 😄

**Review by calumryan — APPROVED** [review]

File: guides/learning-and-development.md (hunk 1 above)
Comment (RobjS) [reply guides/learning-and-development.md]: Updated 👍

File: guides/learning-and-development.md (hunk 3 above)
Comment (RobjS) [reply guides/learning-and-development.md]: 👍

**Review by deberny — APPROVED** [review]

**Comment by CristinaRO** [comment]
@RobjS Did you have a chance to look over the additions Agz suggested? If you don't have time for them, the PR is approved, so feel free to merge.

**Comment by RobjS** [comment]
I was hoping @vanessa-w could take a look before merging - Vee, please could you review when you have the chance?

**Pull request merged**